### ISSUE TITLE
"Account data" button in Explore Account Data is scary and unclear

### ISSUE THREAD

**Issue by ShadowJonathan** [vector-im/element-web#18966]
### What happened?

When checking out Account Data, the below button looked like a destructive operation to me;

![image](https://user-images.githubusercontent.com/22740616/132694110-cac0a1f0-6005-466c-b49c-cddec5157fa8.png)

So i didnt dare to click it, only after I looked at it via devtools (browser) and saw that no text was chopped off, did i click it.

The fact it's a toggle button goes completely against any design sense i have, so much so that even if the button was green, i wouldn't know how to switch to room data.

I'd say it has to be replaced with "switch to room data" and "switch to account data" buttons respectively.

**Comment by t3chguy** [comment]
Duplicate of https://github.com/vector-im/element-web/issues/9583
